src/Forms/__snapshots__/Forms.test.js.snap

```diff
@@ -33,7 +33,7 @@ exports[`<Forms /> create form item 1`] = `
       *
     </label>
     <input
-      className="fd-form__control is-help"
+      className="fd-form__control is-warning"
```

Comment:
just to clarify - `is-help` is not a fundamentals style if you check the markup here: https://sap.github.io/fundamental/components/form.html#input-help-elements

Reply:
nice catch, @jbadan